src/integrationTest/java/uk/gov/hmcts/darts/notification/helper/TemplateIdHelperTest.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.darts.notification.helper;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.darts.notification.exception.TemplateNotFoundException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest
+@ActiveProfiles({"dev","test"})
+class TemplateIdHelperTest {
+
+    @Autowired
+    TemplateIdHelper templateIdHelper;
+
+    @Test
+    void okCourtManagerApproveTranscript() throws TemplateNotFoundException {
+        String templateId = templateIdHelper.findTemplateId("court_manager_approve_transcript");
+        assertEquals("2f9e18b5-5046-44c2-8abd-e4ff15008a60", templateId);
```

Comment:
Any reason this is in the integration tests? Shouldn't it just be at the unit test level?

Reply:
i had it as a unit test earlier, and someone told me to move it to integration because it was making use of springBoot

Reply:
moved back